### PR TITLE
ui updates and misc fixes

### LINKED ISSUE #787: Minor UI issues
The font on this has gone strange too...

Screen Shot 2022-02-09 at 2 49 34 PM

Screen Shot 2022-02-09 at 2 49 43 PM

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #787 

really just a collection of fixes for things what I've broken 🙄 

The strange one, is the change in `Portal` which is coming from `@mui/base` and currently breaking the demo server.
Have added in item unit to the inbound edit modal, fixed styles and corrected a couple of mistakes I made in the android PR

### CODE REVIEW

File: packages/android/build_remote_server_libs.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Path the the remote server dir
-REMOTE_SERVER_DIR="../../remote-server"
+REMOTE_SERVER_DIR="../../../remote-server"
```
Comment (mark-prins) [inline packages/android/build_remote_server_libs.sh:5]: realised this one early this morning 🤦

File: packages/common/src/styles/theme.ts [hunk 2]
```diff
@@ -190,8 +191,9 @@ export const themeOptions = {
       lineHeight: 1.71,
       color: '#1c1c28',
     },
-    body2: { color: '#fafafa', fontSize: 20, fontWeight: 600 },
+    body2: { color: '#555770', fontSize: 12, fontWeight: 500 },
```
Comment (mark-prins) [inline packages/common/src/styles/theme.ts:194]: another mistake causing a few stylistic problems. have extended `Typography` instead

File: packages/common/src/ui/components/buttons/ToggleButton.tsx [hunk 3]
```diff
@@ -27,6 +27,7 @@ export const ToggleButton: FC<ToggleButtonProps> = ({
     sx={{
       backgroundColor: 'white',
       '&.Mui-selected': { backgroundColor: 'white' },
+      '& .MuiSvgIcon-root': { width: '15px', height: '15px' },
```
Comment (mark-prins) [inline packages/common/src/ui/components/buttons/ToggleButton.tsx:30]: was looking a bit big to me

File: packages/common/src/ui/forms/Modal/ModalLabel.tsx [hunk 4]
```diff
@@ -21,11 +21,11 @@ export const ModalLabel: React.FC<ModalLabelProps> = ({
     xs={2}
     alignItems="center"
     justifyContent={justifyContent}
-    flexBasis={0}
     sx={{
       alignItems: 'center',
       display: 'flex',
       minWidth: '80px',
+      '&.MuiGrid-root': { flexBasis: 0 },
```
Comment (mark-prins) [inline packages/common/src/ui/forms/Modal/ModalLabel.tsx:28]: it was making the autocomplete row label too wide

File: packages/host/src/components/Login/hooks.ts [hunk 5]
```diff
@@ -69,7 +74,8 @@ export const useLoginForm = (
     if (store) setHostStore(store);
     // navigate back, if redirected by the <RequireAuthentication /> component
     // or to the dashboard as a default
-    const from = location.state?.from?.pathname || `/${AppRoute.Dashboard}`;
+    const state = location.state as State | undefined;
```
Comment (mark-prins) [inline packages/host/src/components/Login/hooks.ts:77]: just to get the `pre-commit-lint` to pass

File: packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEditForm.tsx [hunk 6]
```diff
@@ -46,23 +46,13 @@ export const InboundLineEditForm: FC<InboundLineEditProps> = ({
...
-            flex={1}
-          >
-            <ModalLabel label={t('label.unit')} justifyContent="flex-end" />
-            <BasicTextInput
-              disabled
-              sx={{ width: 150 }}
-              value={item.unitName}
-            />
-          </Grid>
+        <ModalRow margin={3}>
```
Comment (mark-prins) [inline packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEditForm.tsx:49]: made it like the outbound version, where it was requested that the code be dropped as a separate field

**Review by joshxg — APPROVED** [review]
Nice looking slick!

**Pull request merged**